cmd/ttn-lw-cli/commands/utils.go

```diff
@@ -111,3 +111,27 @@ func parsePayloadFormatterParameterFlags(prefix string, formatters *ttnpb.Messag
 	}
 	return util.NormalizePaths(paths), nil
 }
+
+// usageTemplateWithoutGlobalFlags is the default usage message from Cobra, with
+// the Global Flags section stripped away.
```

Comment:
Why?

Reply:
Global flags are ignored in the `use` command, and may be misleading for the users.

We can just revert if this does not help.

Reply:
I agree that some of those global flags can be confusing, but then I'd prefer fixing that for all commands in a separate PR.